mod-orders/schemas/composite_purchase_order.json

```diff
@@ -122,21 +116,20 @@
     },
     "vendor": {
       "description": "UUID of the vendorDetails record",
-      "type": "string",
       "$ref": "../../common/schemas/uuid.json"
     },
     "workflowStatus": {
       "description": "the workflow status for this purchase order",
       "type": "string",
       "$ref": "../../mod-orders-storage/schemas/workflow_status.json"
     },
-    "compositePoLines": {
-      "description": "a list of completely de-referenced purchase order lines",
-      "id": "compositePoLines",
+    "poLines": {
```

Comment:
also samples need to be updated https://github.com/folio-org/acq-models/blob/0e00eadb5ba456ce763a011fd3634a744d0f49b0/mod-orders/examples/composite_purchase_orders.sample#L45

Reply:
also this https://github.com/folio-org/acq-models/blob/4da19b184216d4b2803bc05a5eb2d10b18755949/mod-orders/examples/composite_purchase_order.sample#L47

Reply:
Thanks, somehow I missed these ones.